crates/nomination/src/lib.rs

```diff
@@ -199,6 +204,21 @@ pub mod pallet {
             Self::_withdraw_collateral(&vault_id, &nominator_id, amount, index.unwrap_or_default())?;
             Ok(().into())
         }
+
+        #[pallet::weight(<T as Config>::WeightInfo::set_nomination_limit())]
+        #[transactional]
+        pub fn set_nomination_limit(
+            origin: OriginFor<T>,
+            currency_pair: DefaultVaultCurrencyPair<T>,
+            limit: BalanceOf<T>,
+        ) -> DispatchResultWithPostInfo {
+            ext::security::ensure_parachain_status_running::<T>()?;
```

Comment:
Probably doesn't matter too much that the parachain is running given that this is a simple storage setter and the call would anyway get filtered during shutdown but I also see no reason not to have this check

Reply:
Agreed. It got added because I copied & modified the other setter. Let's keep it for now for consistency